transaction-scenarios/generated-examples/fungible_resource/1--fungible-max-div-create.rtm

```diff
@@ -0,0 +1,51 @@
+CALL_METHOD
```

Comment:
The ordering of these has a wonky order, maybe we should left-pad the numbers in the file names, eg `001--`?

Reply:
Good idea!